multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -437,6 +455,8 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 		newCondition.Message = getStringPointer("The Service is imported, not allowed to export")
 	case serviceExported:
 		newCondition.Status = corev1.ConditionTrue
+		newCondition.Reason = getStringPointer("Succeed")
+		newCondition.Message = getStringPointer("The Service is exported successfully")
```

Comment:
Note: Reason and Message are required fields in upstream ServiceExport definition, so recover it to fix the bug which is introduced by #4814.